WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -32315,6 +32315,25 @@
    culture="old_frankish"
    religion=nicene
 }
+1044124=
+{  
+   name="Nicetian"
+   culture="romano_gallic"
+   religion=nicene
+}
+1044125=
```

Comment:
You seem to have changed the ID of the Agilolfing dynasty.

Reply:
That has been solved, don´t worry, I changed the ID of the characters that holded the previous dynasty ID. I did that to order my things better

Reply:
I'd rather keep the vanilla numbers for compatibility if we are to integrate vanilla characters etc.